interface/billing/billing_process.php

```diff
@@ -60,6 +60,13 @@
     </div>
 </html>
 <?php
-    $logger->onLogComplete();
+$logger->onLogComplete();
+$hlog = $logger->hlog();
+if (!empty($hlog)) {
+    if ($GLOBALS['drive_encryption']) {
+        $hlog = $cryptoGen->encryptStandard($hlog, null, 'database');
+    }
+    file_put_contents($GLOBALS['OE_SITE_DIR'] . "/documents/edi/process_bills.log", $hlog);
+}
```

Comment:
yep, being able to actually write to the log is a useful thing to support :)

Reply:
I would probably put this in the BillingProcessor class but we can always clean up later.